Assignment/Homework_1 - HTML/pokedex.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Pokedex</title>
+    </head>
+
+    <body>
+        <header>
+            <nav id="navigation">
+                <a class="nav-button"> <button> Dropdown </button> </a>
+                <a class="nav-button"> <b> Pokedex </b> </a>
+                <img src = "/Users/satishkakkar/Downloads/assets/icons/pokeball_icon.png">
```

Comment:
When you add your local file path to the src or href, such as "/Users/satishkakkar/Downloads/", the user will not be able to see the images or redirect to the hyperlink page because the user does not have this file path. Since your files were in the same folder it would have been better to just use "assets/icons/pokeball_icon.png", and in this case the user/we would have been able to see the images